README.md

```diff
@@ -106,7 +106,7 @@ This project is backed by [Deltablot](https://www.deltablot.com) company.
   </a>
 </p>
 
-* Atelier de la donnée ADOC Lorraine (projet financé avec le soutien du Fonds national pour la Science ouverte)
+* Atelier de la donnée ADOC Lorraine (project financé avec le soutien du Fonds national pour la Science ouverte)
```

Comment:
this is a french sentence. Is it possible to tell codespell to avoid this word only in this file?

Reply:
Only an ugly way by listing an entire line verbatim to ignore. Eg we do in https://github.com/datalad/datalad/blob/maint/.codespell-ignorelines .
Easier to just ignore this word everywhere. Which way would you prefer?

Reply:
And sorry about not catching this

Reply:
done, pushed